contrib/scoverage/src/mill/contrib/scoverage/ScoverageReportWorker.scala

```diff
@@ -35,10 +35,7 @@ class ScoverageReportWorker {
       )(implicit
           ctx: TaskCtx
       ): Unit = {
-        mill.util.Jvm.withClassLoader(
-          classpath.map(_.path).toVector,
-          getClass.getClassLoader
-        ) { cl =>
+        mill.util.Jvm.withClassLoader(classpath.map(_.path).toVector) { cl =>
```

Comment:
This looks suspicious. We should not refactor an explicitly known classloader into some implicitly known. This reduces clarity. Although there is a concept of a context class loader, we should not rely on it if we can manage it directly.